app/views/transactions/_form.html.slim

```diff
@@ -63,4 +63,7 @@
   = f.input :date, as: :string, placeholder: 'Date',
     input_html: { class: 'datepicker', value: f.object.date.present? ? l(f.object.date.to_date) : nil }
 
+  - if @invoice.present? || f.object.invoice_id
+    = link_to "Created from invoice", @invoice, class: 'pull-right'
```

Comment:
@dissident I'm not sure if we have `@invoice` on edit existing transaction. Please check it and fix. We don't need to show link to Invoice on create form, only when edit existing transactions.